pwem/protocols/protocol_sets.py

```diff
@@ -139,13 +140,19 @@ def createOutputStep(self):
 
         # Read ClassName and create the corresponding EMSet (SetOfParticles...)
         try:
-            outputSetFunction = getattr(self, "_create%s" % set1.getClassName())
+            if str(set1.getClassName()) is not 'Volume':
+                outputSetFunction = getattr(self, "_create%s" % set1.getClassName())
+            else:
+                outputSetFunction = getattr(self, "_createSetOfVolumes")
```

Comment:
Can't this be replaced by "self._createSetOfVolumes" ?

Reply:
Done

Reply:
Thanks @ajimoreno I've added a test for this case